### ISSUE TITLE
Update documentation for JSNLog RequestId in NLog on AspNetCore

### ISSUE THREAD

**Issue by snakefoot** [mperdeck/jsnlog#179]
Maybe consider using this for extracting RequestId in NLog (Skips need for hook into OnLogging and updating NLog MDC):

`<target layout="${aspnet-request:header=JSNLog-RequestId:whenEmpty=${aspnet-TraceIdentifier}}" />`

http://jsnlog.com/Documentation/HowTo/RequestIds#nlog

Requires include of https://www.nuget.org/packages/NLog.Web.AspNetCore as extension:

```
<extensions>
  <add assembly="NLog.Web.AspNetCore"/>
</extensions>
```

See also https://stackoverflow.com/questions/53579396/configure-jsnlog-with-nlog-for-requestid-logging-under-asp-net-core

**Comment by snakefoot** [comment]
See also https://github.com/mperdeck/jsnlogSimpleWorkingDemos/pull/3

**Comment by axunonb** [comment]
Thanks for upvoting the PR.
Indeed, integration for NLog is more simple than JsNLog docs currently say.

**Comment by mperdeck** [comment]
Thanks for the heads up. This has now been released as part of version 2.29.0.
